### ISSUE TITLE
buying train tickets??

### ISSUE THREAD

**Issue by Hayleene** [Hayleene/train-timetable#1]
We have been trying to purchase train tickets on an German website. We are trying to get from Berlin to Barcelona on July 18, and there is a train listed for 9:30 but though we have checked back many times over the last months, it says that tickets are not available and to check back later. It has been suggested that the tickets may not be available until 1 month before that date. Is that true/possible? What have others experienced in similar situations? Many thanks.

**Comment by MikeColady** [comment]
> We have been trying to purchase train tickets on an German website. We are trying to get from Berlin to Barcelona on July 18, and there is a train listed for 9:30 but though we have checked back many times over the last months, it says that tickets are not available and to check back later. It has been suggested that the tickets may not be available until 1 month before that date. Is that true/possible? What have others experienced in similar situations? Many thanks.

Yes, it's possible that tickets for your desired train from Berlin to Barcelona might not be available until closer to the departure date, commonly within a month. This is often the case with certain routes. For checking timetables and schedules, you can use [DB Reiseauskunft](https://www.dbauskunft.com/de/).
